magicctapipe/scripts/lst1_magic/README.md

```diff
@@ -31,6 +31,215 @@ MAGIC+LST-1 analysis starts from MAGIC calibrated data (\_Y\_ files), LST-1 DL1
 - `lst1_magic_create_irf.py` to create the IRF
 - `lst1_magic_dl2_to_dl3.py` to create DL3 files, and `create_dl3_index_files.py` to create DL3 HDU and index files
 
-## High level analysis
+## MAGIC+LST analysis: data reduction tutorial (PRELIMINARY)
+
+1) The very first step to reduce MAGIC-LST data is to have remote access/credentials to the IT Container, so provide one. Once you have it, the connection steps are the following:  
+
+Authorized institute server (Client) &rarr;  ssh connection to CTALaPalma &rarr; ssh connection to cp01/02  
```

Comment:
we are used to do things in IT container, but the pipeline should not be fixed on particular computing system. We already now have PIC plus 2 auxiliary analysis centers and the bulk of the analysis will sooner or later move there to lessen the burden from IT container on things that really need to be done there.

Reply:
We moved the IT-scripts (the ones creating bash scripts) to a sub-folder, and other sub-folders could be created if/when needed to store the scripts that produce the bash scripts used by other clusters (PIC, CNAF,....). Different computing systems necessarily require a different bash script structure and different job submission commands, so the pipeline is and will be cluster-independent, but the scripts that automatize it depend on the used analysis center.

Reply:
Ok! This README is now available only in the Torino_IT_Container branch.